Prace_domowe/Praca_domowa2/Grupa4/Marcin Wilk/PD2.ipynb

```diff
@@ -0,0 +1,1563 @@
+{
```

Comment:
<p>Tylko, żeby nie były to 2 lata świetlne</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>